scout/adapter/utils.py

```diff
@@ -8,19 +8,20 @@
     from urllib import quote_plus
 
 from pymongo import MongoClient
-from pymongo.errors import OperationFailure, ServerSelectionTimeoutError
+from pymongo.errors import ConnectionFailure, OperationFailure, ServerSelectionTimeoutError
 
 LOG = logging.getLogger(__name__)
 
 
 def check_connection(
-    host="localhost", port=27017, username=None, password=None, authdb=None, max_delay=1
+    host="localhost", port=27017, uri=None, username=None, password=None, authdb=None, max_delay=1
```

Comment:
👍 
I guess we really wont use it any more, except for in some old scripts. Good to fix if its around, but we should probably deprecate it as well.